Technical Docs for our Server tests setup with TestNG are outdated

The download link in our docs for TestNG update URL is now out of date and some of the info in this section:  https://docs.openrefine.org/technical-reference/build-test-run#testing-in-eclipse

There's good info to get and reuse some of it from their README.md file in their repo here:
https://github.com/cbeust/testng-eclipse

Error: Unable to initialize main class org.testng.remote.RemoteTestNG
Caused by: java.lang.NoClassDefFoundError: org/testng/TestNGException

I encounters this error which seems to be related to the classpath settings for TestNG, according to the official doc https://www.testingdocs.com/questions/how-to-fix-the-error-unable-to-initialize-main-class-org-testng-remote-remotetestng/

However, I cannot add the classpath according to the instructions. The "Add Libray..." button is unselectable.
![image](https://user-images.githubusercontent.com/102639466/161346671-9de7b141-ecbe-401c-903f-135bbebe4454.png)

Which seems to be related with build path entry is missing: server/src/main/java
![image](https://user-images.githubusercontent.com/102639466/161346817-d6985513-78db-4425-bebf-516632291e62.png)

I don't have lots of experience with maven setup / plugins in Eclispe. Would appreciate any help from here. Thanks.

--- a/docs/docs/technical-reference/build-test-run.md
+++ b/docs/docs/technical-reference/build-test-run.md
@@ -256,7 +256,7 @@ You can manually apply the code style (regardless of your IDE) with the `mvn for
 
 ## Testing in Eclipse {#testing-in-eclipse}
 
-You can run the server tests directly from Eclipse. To do that you need to have the TestNG launcher plugin installed, as well as the TestNG M2E plugin (for integration with Maven). If you don't have it, you can get it by [installing new software](https://help.eclipse.org/2020-03/index.jsp?topic=/org.eclipse.platform.doc.user/tasks/tasks-129.htm) from this update URL http://dl.bintray.com/testng-team/testng-eclipse-release/
+You can run the server tests directly from Eclipse. To do that you need to have the TestNG launcher plugin installed, as well as the TestNG M2E plugin (for integration with Maven). If you don't have it, you can get it by [installing new software](https://help.eclipse.org/2020-03/index.jsp?topic=/org.eclipse.platform.doc.user/tasks/tasks-129.htm) from this update URL https://testng.org/doc/download.html
 
 Once the TestNG launching plugin is installed in your Eclipse, right click on the source folder "main/tests/server/src", select `Run As` -> `TestNG Test`. This should open a new tab with the TestNG launcher running the OpenRefine tests.
 

